Marlin/Marlin_main.cpp

```diff
@@ -6045,6 +6084,29 @@ inline void gcode_M121() { endstops.enable_globally(false); }
 
 #endif // BLINKM || RGB_LED
 
+#if ENABLED(LED_STRIP)
+
+  /**
+   * M151: Set RGB LED Strip Color - Use R-U-B for R-G-B
+   *
+   * Always sets all 3 components. If a component is left out, set to 0.
+   *
+   * Examples:
+   *
+   *   M150 R255       ; Turn LED Strip to red
+   *   M150 R255 U127  ; Turn LED Strip orange
+   *   M150            ; Turn LED Strip off
+   *   M150 R U B      ; Turn LED Strip white
+   *
+   */
+  inline void gcode_M151() {
+    code_seen('R') ? (code_has_value() ? code_value_byte() : 255) : 0,
+    code_seen('U') ? (code_has_value() ? code_value_byte() : 255) : 0,
+    code_seen('B') ? (code_has_value() ? code_value_byte() : 255) : 0
+  );
```

Comment:
Oops. This is missing the function call.

You can actually overload `M150` instead of adding `M151`.